slider/replay.py

```diff
@@ -25,8 +25,10 @@ class Action:
 
     Parameters
     ----------
-    offset : timedelta
+    rel_offset : timedelta
```

Comment:
I hadn't thought of comparing 2 offsets to determine change, in that case I'd agree with removing relative offset.